src/app/Plans/components/Wizard/WizardComponent.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import WizardStepContainer from './WizardStepContainer';
+import { Wizard } from '@patternfly/react-core';
+import Review from './Review';
+import GeneralForm from './GeneralForm';
+import { Provider } from '@app/Providers/types';
+
+interface IWizardComponent {
```

Comment:
For consistency, let's put `Props` at the end of this interface

Reply:
Sorry, I know better.